build/preset-env.js

```diff
@@ -26,6 +26,7 @@ const browsers = [
 
 const envOptions = {
   targets: { browsers },
+  useBuiltIns: 'entry',
```

Comment:
This change will affect also `dist` build that will be published to NPM. We also using [transform runtime](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/.babelrc#L16).

I am not sure that proposed changes are safe.

Reply:
I'm not confident that this setting helps the docs. Albeit crude, my tests showed no difference in network traffic. I'll take it out for now.